src/rootscripts/plotGui.C

```diff
@@ -318,150 +337,199 @@ void plotGuiMainFrame::PrintSelected(const int sel1, const int sel2)
+				continue;
+			}
 
-		if((numeric_limits<double>::has_infinity and intensity2 == numeric_limits<double>::infinity()) or intensity2!=intensity2) {
-			continue;
-		}
+			g1->SetPoint(i, result->massBinCenter()*0.001, intensity1);
+			g1->SetPointError(i, _binWidth*0.5, intensityErr1);
 
-		g2->SetPoint(i, result->massBinCenter()*0.001, intensity2);
-		g2->SetPointError(i, _binWidth*0.5, intensityErr2);
-
-		// check if we should make a transformation by 2pi
-		// this is needed because of cyclical variable phi
-		if(i>11){
-			double mpre;
-			double phpre;
-			gph->GetPoint(i-1,mpre,phpre);
-			double diff1=fabs(ph-phpre);
-			double diff2=fabs(ph+360-phpre);
-			double diff3=fabs(ph-360-phpre);
-			if(diff2<diff1 && diff2<diff3)ph+=360;
-			else if(diff3<diff1 && diff3<diff2)ph-=360;
-		}
+			if((numeric_limits<double>::has_infinity and intensity2 == numeric_limits<double>::infinity()) or intensity2!=intensity2) {
+				continue;
+			}
 
-		gph->SetPoint(i, result->massBinCenter()*0.001, ph);
-		gph->SetPointError(i, _binWidth*0.5, phErr);
+			g2->SetPoint(i, result->massBinCenter()*0.001, intensity2);
+			g2->SetPointError(i, _binWidth*0.5, intensityErr2);
+
+			// check if we should make a transformation by 2pi
+			// this is needed because of cyclical variable phi
+			if(i>11){
+				double mpre;
+				double phpre;
+				gph->GetPoint(i-1,mpre,phpre);
+				double diff1=fabs(ph-phpre);
+				double diff2=fabs(ph+360-phpre);
+				double diff3=fabs(ph-360-phpre);
+				if(diff2<diff1 && diff2<diff3)ph+=360;
+				else if(diff3<diff1 && diff3<diff2)ph-=360;
+			}
 
-		// add point +- 360 degree
-		gphP1->SetPoint(i, result->massBinCenter()*0.001, ph+360);
-		gphP1->SetPointError(i, _binWidth*0.5, phErr);
+			gph->SetPoint(i, result->massBinCenter()*0.001, ph);
+			gph->SetPointError(i, _binWidth*0.5, phErr);
 
-		gphM1->SetPoint(i, result->massBinCenter()*0.001, ph-360);
-		gphM1->SetPointError(i, _binWidth*0.5, phErr);
+			// add point +- 360 degree
+			gphP1->SetPoint(i, result->massBinCenter()*0.001, ph+360);
+			gphP1->SetPointError(i, _binWidth*0.5, phErr);
 
-		gRe->SetPoint(i, result->massBinCenter()*0.001, rho.real());
-		gRe->SetPointError(i, _binWidth*0.5, sqrt(rhoCov[0][0]));
-		gIm->SetPoint(i, result->massBinCenter()*0.001, rho.imag());
-		gIm->SetPointError(i, _binWidth*0.5, sqrt(rhoCov[1][1]));
-	}// end loop over bins
+			gphM1->SetPoint(i, result->massBinCenter()*0.001, ph-360);
+			gphM1->SetPointError(i, _binWidth*0.5, phErr);
+
+			gRe->SetPoint(i, result->massBinCenter()*0.001, rho.real());
+			gRe->SetPointError(i, _binWidth*0.5, sqrt(rhoCov[0][0]));
+			gIm->SetPoint(i, result->massBinCenter()*0.001, rho.imag());
+			gIm->SetPointError(i, _binWidth*0.5, sqrt(rhoCov[1][1]));
+		}// end loop over bins
+
+	intensity1Graph->Add(g1);
+	intensity2Graph->Add(g2);
+	phaseGraph->Add(gph);
+	phaseGraph->Add(gphP1);
+	phaseGraph->Add(gphM1);
+	reGraph->Add(gRe);
+	imGraph->Add(gIm);
```

Comment:
Are these whitespace errors, or is this a problem with the display in a web browser?